ui/src/api/login.js

```diff
@@ -0,0 +1,23 @@
+import api from '.';
+
+const endpoint = api.url('/login');
+
+export function logIn(proposal, password, previousUser) {
+  return endpoint.post({ proposal, password, previousUser }, '/').json();
+}
+
+export function signOut() {
+  return endpoint.headers({ Accept: '*/*' }).get('/signout').res();
```

Comment:
I meant to ask you about this `signout` endpoint @marcus-oscarsson. I've noticed that it [redirects to the `login` endpoint](https://github.com/mxcube/mxcubeweb/blob/7f9159b20f4e440c8123b4f285cdc74599ec5e71/mxcubeweb/routes/login.py#L70), which seems very weird and requires me to "unset" the `Accept: 'application/json` header. Shouldn't it just respond with a 200?

Reply:
I see, I agree that the redirect is probably unnecessary. I'm not sure why its there. I suggest that we simply try without it :)

Reply:
I think it's trying to behave like a multi-page app by telling the browser to redirect to `/logout`, which obviously doesn't work since we use `fetch`.